[2505] Updated documentation of the ROS2PoseControl (#134)
RobotecAI/robotec-o3de-tools: readme.md

# ROS2PoseControl

> A utility gem that introduces a way to control robots in simulation as puppets with Pose messages or TFs. For a more detailed description, check the [PoseControl user guide](./doc/UserGuide/PoseControl.md).

Reviewer: ```suggestion
The utility gem enabling controlling robots in simulation as puppets with Pose messages or TFs. For a more detailed description, check the [PoseControl user guide](./doc/UserGuide/PoseControl.md).
```
